.circleci/docker/build.sh

```diff
@@ -277,6 +277,14 @@ docker build \
        "$@" \
        .
 
+# NVIDIA dockers for RC releases use tag names like `11.0-cudnn8-devel-ubuntu18.04-rc`,
+# for this case we will set UBUNTU_VERSION to `18.04-rc` so that the Dockerfile could
+# find the correct image. As a result, here we have to replace the
+#   "$UBUNTU_VERSION" == "18.04-rc"
+# with
+#   "$UBUNTU_VERSION" == "18.04"
+UBUNTU_VERSION=$(echo ${UBUNTU_VERSION} | grep -oP '\d*\.\d*$|\d*\.\d*(?=-rc)')
```

Comment:
Would it be better here to just strip any instance of `-rc` here?

```suggestion
UBUNTU_VERSION=$(echo ${UBUNTU_VERSION} | sed 's/-rc$//')
```

Reply:
Applied and this PR should be ready now